daemon/api.go

```diff
@@ -585,21 +585,50 @@ func deleteOp(c *Command, r *http.Request) Response {
 	}
 }
 
+type licT struct {
+	Intro   string
+	License string
+	Agreed  bool
+}
+
+func (*licT) Error() string {
+	// licT isn't an error, strictly speaking, but can
+	// be converted into one for convenience
```

Comment:
The data is sent to the client, structured, as tested for in `TestInstallLicensed` and as I showed you in http://pastebin.ubuntu.com/13237922/.